docs/src/main/paradox/security.md

```diff
@@ -0,0 +1,26 @@
+# Security announcements
+
+## Receiving security advisories
+The best way to receive any and all security announcements is to subscribe to the [Akka security list](https://groups.google.com/forum/#!forum/akka-security).
+
+The mailing list is very low traffic, and receives notifications only after security reports have been managed by the core team and fixes are publicly available.
+
+## Reporting vulnerabilities
+
+We strongly encourage people to report such problems to our private security mailing list first, before disclosing them in a public forum.
+
+Following best-practice, we strongly encourage anyone to report potential security 
+vulnerabilities to security@lightbend.com before disclosing them in a public forum like the mailing list or as a Github issue.
+
+Reports to this email address will be handled by our security experts with priority. We'll work together with you
+to ensure that a fix can be provided without delay.
+
+## Fixed Security Vulnerabilities
+
+### Fixed in Akka HTTP 2.4.11
+
+@@@ index
+
+- [Directory Traversal Vulnerability Announcement](security/2016-09-30-windows-directory-traversal.md)
```

Comment:
Will internal .md links work?  I've had to quote them in `[[internal]]` link style to work when used outside of the table of contents.